app/src/main/java/com/name/name/musicappmvp/data/repository/LocalSongRepository.kt

```diff
@@ -0,0 +1,26 @@
+package com.name.name.musicappmvp.data.repository
+
+import com.name.name.musicappmvp.data.model.LocalSong
+import com.name.name.musicappmvp.data.source.LocalSongDataSource
+import com.name.name.musicappmvp.data.source.OnGotListCallback
+import java.lang.Exception
+
+class LocalSongRepository(private val localSongDataSource: LocalSongDataSource.Local) :
+    LocalSongDataSource.Local {
+
+    override fun getLocalSong(callback: OnGotListCallback): MutableList<LocalSong> {
```

Comment:
giá trị trả về để làm gì nhỉ? 
```suggestion
    override fun getLocalSong(callback: OnGotListCallback): MutableList<LocalSong> {
```